uniqueidentifier.go

```diff
@@ -48,6 +48,9 @@ func (u *UniqueIdentifier) Scan(v interface{}) error {
 
 		_, err := hex.Decode(u[:], []byte(b))
 		return err
+	case nil:
+		*u = [16]byte{0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0}
+		return nil
```

Comment:
The alternative would be to implement a [NullUUID](https://github.com/google/uuid/blob/master/null.go#L29) like [google/uuid](https://github.com/google/uuid) which would keep the existing behavior of erroring out on `NULL` values in the database. The downside of this is that we would have to add another type.